jdaviz/configs/imviz/helper.py

```diff
@@ -152,11 +152,51 @@ def link_data(self, **kwargs):
         """
         link_image_data(self.app, **kwargs)
 
+    def load_static_regions_from_file(self, region_file, region_format='ds9', prefix='region',
+                                      max_num_regions=20, **kwargs):
```

Comment:
@orifox , it limits to first 20 regions in the file on load but does not check if total loaded Subsets are 20, as we discussed offline.